assets/_variables.scss

```diff
@@ -7,6 +7,7 @@ $cd-purple-gradient: #9266a6;
 $cd-orange: #ED684A;
 $cd-orange-gradient: #EC431D;
 $cd-teal: #41BAC1;
+$foam: #D1FAFD;
```

Comment:
For consistency, this should possibly be '$cd-foam', but don't know why we prefix the colours anyway - it's a minor thing and doesn't need fixing unless you want to start doing all of the other things that need fixing too :)